src/controllers/dlgprefcontroller.cpp

```diff
@@ -832,6 +870,35 @@ void DlgPrefController::initTableView(QTableView* pTable) {
     pTable->setAlternatingRowColors(true);
 }
 
+#ifdef MIXXX_USE_QML
+void DlgPrefController::slotShowPreviewScreens(
```

Comment:
are preview screens the same as Dummy screens? if so, we should use one name for them. If not, we need more comments about what the difference is

Reply:
It looks like "preview" screens may just be the screens on the controller.  why "preview"? why not just "ControllerScreen"?

Reply:
No - "preview screen" allows you to mirror the screen in the preference settings, which can be used if you want to test or debug screen design without using the device (preventing wear and tear, improving DX, ...)

"Dummy screens" are just screens I've made to test the various features (different form factor, different pixel format, refresh rate, ...) which are "vendor agnostic" (not related to S4 Mk3, meant to be runnable by everyone who want to test this feature)

I'm planning to remove "Dummy screen" entirely once everyone had a chance to test with then and fully reviewed the PR. `ControllerScreenPreview` is a widget that allow to mirror screens in the UI, which helps with custom mapping, debugging or also sharing look-n-feel (e.g bug report can be done with a screen shot - even maybe a "save" button in the future? - instead of taking a picture of the device)